### PR TITLE
Cpma e2e report testing.

### PR DESCRIPTION
**Pull request by Danil-Grigorev**
Introducing base pipeline for CPMA functional testing.

### CODE REVIEW

File: pipeline/cpma-base.groovy [hunk 1]
```diff
@@ -0,0 +1,47 @@
+properties([
+parameters([
+credentials(credentialType: 'org.jenkinsci.plugins.plaincredentials.impl.FileCredentialsImpl', defaultValue: 'ci_ec2_key', description: 'EC2 private key needed to access instances, from Jenkins credentials store', name: 'EC2_PRIV_KEY', required: true),
+string(defaultValue: 'ci', description: 'EC2 SSH key name for remote access', name: 'EC2_KEY', trim: false),
+string(defaultValue: 'HEAD', description: 'CPMA branch to checkout', name: 'CPMA_BRANCH', trim: false),
+string(defaultValue: 'fusor/cpma.git', description: 'CPMA repo to clone', name: 'CPMA_REPO', trim: false),
+string(description: 'CPMA repo to clone', name: 'CPMA_PASSWD', trim: false),
```
Comment (fbladilo) [inline pipeline/cpma-base.groovy]: Descriptions need to be accurate for all these parameters

**Comment by fbladilo** [comment]
@Danil-Grigorev Just to confirm, this CPMA e2e pipeline is assuming a cluster to be already provisioned by some other means?

**Comment by Danil-Grigorev** [comment]
Yes, the cluster should be already provisioned. For this purpose there is a stack of stable clusters on eu-central-1 region now.

File: pipeline/cpma-base.groovy [hunk 2]
```diff
@@ -0,0 +1,47 @@
+properties([
+parameters([
+credentials(credentialType: 'org.jenkinsci.plugins.plaincredentials.impl.FileCredentialsImpl', defaultValue: 'ci_ec2_key', description: 'EC2 private key needed to access instances, from Jenkins credentials store', name: 'EC2_PRIV_KEY', required: true),
+string(defaultValue: 'ci', description: 'EC2 SSH key name for remote access', name: 'EC2_KEY', trim: false),
+string(defaultValue: 'HEAD', description: 'CPMA branch to checkout', name: 'CPMA_BRANCH', trim: false),
+string(defaultValue: 'https://github.com/fusor/cpma.git', description: 'CPMA repo to clone', name: 'CPMA_REPO', trim: false),
+string(description: 'Cluster hostname for ssh access', name: 'CPMA_HOSTNAME', trim: false),
+string(description: 'Cluster currnet-context master name to generate report from', name: 'CPMA_CLUSTERNAME', trim: false),
```
Comment (fbladilo) [inline pipeline/cpma-base.groovy]: Small typo on current...

**Pull request merged**